README.md

```diff
@@ -97,9 +97,15 @@ Complete all of these tasks on your local home machine.
 * Streisand requires a BSD, Linux, or macOS system. As of now, Windows is not supported. All of the following commands should be run inside a Terminal session.
 * Python 2.7 is required. This comes standard on macOS, and is the default on almost all Linux and BSD distributions as well. If your distribution packages Python 3 instead, you will need to install version 2.7 in order for Streisand to work properly.
 * Make sure an SSH key is present in ~/.ssh/id\_rsa.pub.
+  * SSH keys use public-key cryptography to prove your identity to a server or service.
+  * To check if you already have an SSH key, please enter the following command at a Bash prompt.
```

Comment:
Instead of "at a Bash prompt" maybe it would be better to say something along the lines of "in a terminal at a command prompt". I think most users unfamiliar with SSH keys won't know about specific shells and would be using a GUI to navigate to something like Konsole or another _terminal_ program. What do you think?